### ISSUE TITLE
[Hold][Xerrada] Keras/TensorFlow

### ISSUE THREAD

**Issue by rvinas** [pygrn/xerrades#42]
### Xerrada Keras/TensorFlow

Tipus de la proposta:
- [x] Vull proposar un tema interessant
- [x] Vull fer una xerrada

M'agradaria proposar una xerrada/workshop sobre Keras/TensorFlow. Podríem:

1. Revisar alguns conceptes bàsics de deep learning (i.e. xarxes totalment connectades, convolucionals, recurrents,...). També podríem estudiar algoritmes bàsics per a l'aprenentatge (i.e. backpropagation i gradient descent).
2. Explicar els principis bàsics de Keras/TensorFlow.
3. Fer algun workshop amb algun cas pràctic.

S'admeten altres propostes!

Duració estimada: 1h

TAGS: `Keras, TensorFlow, deep learning, ...`

**Comment by XaviTorello** [comment]
Genial, fa molt bona pinta @rvinas :beers:

**Comment by jbagot** [comment]
Jo proposaria de fer només la xerrada i fer-la de menys duració. I si veiem que hi ha bona acollida podriem programar el workshop per una altre sessió.
Em sembla molt interessant.

**Comment by XaviTorello** [comment]
Em sembla molt bona idea @jbagot, com ho veus @rvinas? Busquem data? ^^

**Comment by victormartingarcia** [comment]
+1 Per només xerrada... workshop per sessions posteriors

**Comment by rvinas** [comment]
Va bé algun dia de la setmana del 18 de febrer?

**Comment by XaviTorello** [comment]
Ens trobem un cop al mes @rvinas, i aquest mes ja la hem fet heheheh

Parlem per privat, però potser podria ser interessant planificar-la per l'abril, que la del març ja la tenim planificada.

Thxs!!

**Comment by jbagot** [comment]
Aquesta xerrada l'hem posat en HOLD degut a raons de disponibilitat del ponent.
Si algú s'anima a fer-la, comenteu-ho :+1:

**Comment by XaviTorello** [comment]
@rvinas, estem estudiant el roadmap d'aquest 2021. Com ho veus si ho plantegem com una sessió online?

**Comment by rvinas** [comment]
@XaviTorello Estaria molt bé... Però ara mateix vaig molt de cul amb el doctorat 😕  Si més endavant tinc temps us aviso
